recipes/libpq/all/conanfile.py

```diff
@@ -160,7 +160,11 @@ def _remove_unused_libraries_from_package(self):
             else:
                 globs = [os.path.join(self.package_folder, "lib", "*.a")]
```

Comment:
Does this work on mingw?
A shared Windows library has a dll and a .a import library.

Reply:
mingw static build doesn't seem to be allowed (in theory): https://github.com/conan-io/conan-center-index/blob/master/recipes/libpq/all/conanfile.py#L57-L58
The condition is wrong. Should I fix it in this PR ?

Reply:
@remiburtin the message is wrong, static is allowed but not shared for mingw

Reply:
@madebr do you confirm this ? I'm not very familiar with mingw. Your commit is a bit ambiguous for me: https://github.com/conan-io/conan-center-index/commit/2f3a50023c27cd3de629f591035f0a34931b0949
Should we handle mingw static/shared issue in this PR ? As mine is specific to mac os.

Reply:
I was just wondering whether this didn't break mingw, because that isn't tested on c3i.
Oh, I committed that myself.
And the message of 2f3a500 is wrong... :man_facepalming: